README.md

```diff
@@ -93,13 +93,13 @@ function* treeWalker(refresh) {
 
     // Basing on the node openness state we are deciding if we need to render
     // the child nodes (if they exist).
-    if (node.children.length !== 0 && isOpened) {
+    if (children && children.length > 0 && isOpened) {
       // Since it is a stack structure, we need to put nodes we want to render
       // first to the end of the stack.
-      for (let i = node.children.length - 1; i >= 0; i--) {
+      for (let i = children?.length - 1; i >= 0; i--) {
```

Comment:
You forgot this one 🙂 It is not necessary to check since we already check it above.